framework/feedback_util.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+import socket
+from subprocess import Popen
+
+
+def start_feedback_server(port):
+    """
+    Start the feedback server as a separate process/thread.
+    Example:
+    --------
+    server_process = start_feedback_server()
+    """
+    if is_port_available(int(port)):
+        logging.info(f"Starting feedback server on port {port}...")
+    else:
+        logging.error(f"Port {port} is already in use. Please choose a different port.")
+    server_process = Popen(
+        ["python3", "../humanFeedback/feedback_server.py", "--port", str(port)]
```

Comment:
```suggestion
        [sys.executable, "../humanFeedback/feedback_server.py", "--port", str(port)]
```
This should ensure it works on every os.